### ISSUE TITLE
Activation function not working on dense layer when imported by tensor name

### ISSUE THREAD

**Issue by Zylatis** [Zylatis/SetNN#1]
When importing metagraph and getting the logits layer by name (graph.get_tensor_by_name) it can give negative numbers showing that the relu function isn't being applied. 

One can show this alongside just referring to the tensor as normal with a session call during training and this remains true (so not the importer). Possibly TF bug?

**Comment by Zylatis** [comment]
Closed as relates to now deprecated TF1 branch
